adsApp/src/adsAsynPortDriver.cpp

```diff
@@ -1130,6 +1160,35 @@ asynStatus adsAsynPortDriver::updateParamInfoWithPLCInfo(adsParamInfo *paramInfo
   return asynSuccess;
 }
 
+void adsAsynPortDriver::poll_info(char *name)
+{
+    int i;
+    printf("Bulk read loop: desired period = %gs, last loop time = %gs\n", bulk_delay_us / 1000000.0, bulk_elapsed_us / 1000000.0);
```

Comment:
Really handy debug info 👍

Reply:
At some point, it would be good to *really* have multiple poll classes instead of just pretending.  But I suppose Rome wasn't built in a day.  Took about two weeks, right? :-)